dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -7271,8 +7271,6 @@ definitions:
         format: "int64"
   ParsedInformation:
     type: "object"
-    required:
```

Comment:
Diito @garyluu not sure why these are changing in ParsedInformation

Reply:
ditto